From 65842732489aa1904e7e04880ff9497a94ed0eb4 Mon Sep 17 00:00:00 2001
From: "Luke A. Guest" <laguest@archeia.com>
Date: Wed, 26 Jan 2022 13:35:51 +0000
Subject: [PATCH 2/2] reapply by hand - add includes for getrlimit

---
 daemon/loader.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/daemon/loader.c b/daemon/loader.c
index 777d7cb..d7b8a21 100644
--- a/daemon/loader.c
+++ b/daemon/loader.c
@@ -34,6 +34,8 @@
 #include <fcntl.h>
 #include <pty.h>                /* forkpty() */
 #include <sys/wait.h>
+#include <sys/time.h>
+#include <sys/resource.h>
 
 #include "loader.h"
 #include "../proxies/conf_proxy.h"
-- 
2.34.1

