--- makeresolvedeb.sh.orig	2022-11-30 17:25:31.889253084 +0000
+++ makeresolvedeb.sh	2022-11-30 17:27:01.019252439 +0000
@@ -60,7 +60,7 @@
     fi
     if [[ -e "${1}" ]] ; then
 	echo "[ERROR: UNABLE TO REPLACE DIRECTORY] $1"
-	ERRORS=$((ERRORS+1))	
+	ERRORS=$((ERRORS+1))
     fi
     create_directory "$1"
 }
@@ -217,7 +217,9 @@
     link_object ../../libs/libBlackmagicRawAPI.so ${RESOLVE_BASE_DIR}/bin/BlackmagicRawAPI/libBlackmagicRawAPI.so
 
     # Create common data dir
+    echo "[process_16] 1"
     create_directory ${DEB_DIR}/var/BlackmagicDesign/DaVinci\ Resolve
+    echo "[process_16] 2"
 
     # Add postinst commands
     cat >> ${DEB_DIR}/DEBIAN/postinst <<EOF
@@ -225,9 +227,11 @@
 chmod -R a+rw /opt/resolve/LUT
 chmod -R a+rw /opt/resolve/.license
 chmod -R a+rw /opt/resolve/Fairlight
+echo "[process_16] 3"
 chmod -R a+rw /var/BlackmagicDesign/"DaVinci Resolve"
+echo "[process_16] 4"
 EOF
-    
+
     # libcudafix for Resolve 16.0 & 16.1
     if [[ "$RESOLVE_VERSION" == 16.0* ]] || [[ "$RESOLVE_VERSION" == 16.1* ]];
     then
@@ -284,7 +288,9 @@
     fi
 
     # Create common data dir
+    echo "[process_17] 1"
     create_directory ${DEB_DIR}/var/BlackmagicDesign/DaVinci\ Resolve
+    echo "[process_17] 2"
 
     # Add postinst commands
     cat >> ${DEB_DIR}/DEBIAN/postinst <<EOF
@@ -292,7 +298,9 @@
 chmod -R a+rw /opt/resolve/LUT
 chmod -R a+rw /opt/resolve/.license
 chmod -R a+rw /opt/resolve/Fairlight
+echo "[process_17] 3"
 chmod -R a+rw /var/BlackmagicDesign/"DaVinci Resolve"
+echo "[process_17] 4"
 EOF
 }
 
@@ -331,14 +339,16 @@
     if [[ -e ${UNPACK_DIR}/share/remote-monitoring-log-conf.xml ]]; then
 	copy_object ${UNPACK_DIR}/share/remote-monitoring-log-conf.xml ${RESOLVE_BASE_DIR}/share/
     fi
-    
+
     # Extract panel API library
     create_directory ${DEB_DIR}/usr/#LIBDIR#/
     extract_tgz ${UNPACK_DIR}/share/panels/dvpanel-framework-linux-x86_64.tgz ${DEB_DIR}/usr/#LIBDIR#/ libDaVinciPanelAPI.so
     extract_tgz ${UNPACK_DIR}/share/panels/dvpanel-framework-linux-x86_64.tgz ${DEB_DIR}/usr/#LIBDIR#/ libFairlightPanelAPI.so
 
     # Create common data dir
+    echo "[process_18] 1"
     create_directory ${DEB_DIR}/var/BlackmagicDesign/DaVinci\ Resolve
+    echo "[process_18] 2"
 
     # Add postinst commands
     cat >> ${DEB_DIR}/DEBIAN/postinst <<EOF
@@ -346,7 +356,9 @@
 chmod -R a+rw /opt/resolve/LUT
 chmod -R a+rw /opt/resolve/.license
 chmod -R a+rw /opt/resolve/Fairlight
+echo "[process_18] 3"
 chmod -R a+rw /var/BlackmagicDesign/"DaVinci Resolve"
+echo "[process_18] 4"
 EOF
 }
 
@@ -356,9 +368,9 @@
     fi
     if [[ -e ${UNPACK_DIR}/BlackmagicRAWPlayer ]]; then
 	echo "Adding BlackmagicRAWPlayer"
-	
+
 	copy_object ${UNPACK_DIR}/BlackmagicRAWPlayer ${RESOLVE_BASE_DIR}
-	
+
 	assert_object ${DEB_DIR}/opt/resolve/graphics/blackmagicraw-player_256x256_apps.png
 	assert_object ${DEB_DIR}/opt/resolve/BlackmagicRAWPlayer/BlackmagicRAWPlayer
 	cat > ${DEB_DIR}/usr/share/applications/${DEB_NAME}-braw-player.desktop <<EOF
@@ -375,12 +387,12 @@
 Categories=AudioVideo
 EOF
     fi
-    
+
     if [[ -e ${UNPACK_DIR}/BlackmagicRAWSpeedTest ]]; then
 	echo "Adding BlackmagicRAWSpeedTest"
-	
+
 	copy_object ${UNPACK_DIR}/BlackmagicRAWSpeedTest ${RESOLVE_BASE_DIR}
-	
+
 	assert_object ${DEB_DIR}/opt/resolve/graphics/blackmagicraw-speedtest_256x256_apps.png
 	assert_object ${DEB_DIR}/opt/resolve/BlackmagicRAWSpeedTest/BlackmagicRAWSpeedTest
 	cat > ${DEB_DIR}/usr/share/applications/${DEB_NAME}-braw-speedtest.desktop <<EOF
@@ -685,7 +697,7 @@
     then
 	ERRORS=$((ERRORS+1))
     fi
-    
+
     # Clean up
     remove_directory ${UNPACK_DIR}
     remove_directory ${DEB_DIR}
